src/sparseml/pytorch/models/classification/efficientnet.py

```diff
@@ -862,4 +1003,334 @@ def efficientnet_b7(
+    def_ignore_error_tensors=["classifier.fc.weight", "classifier.fc.bias"],
+    desc_args={"optim-perf": ("se_mod", True)},
+)
+def efficientnet_v2_s(
+    num_classes: int = 1000,
+    class_type: str = "single",
+    dropout: float = 0.2,
+    se_mod: bool = False,
+) -> EfficientNet:
+    """
+    EfficientNetV2-s implementation; expected input shape is (B, 3, 384, 384)
+
+    :param num_classes: the number of classes to classify
+    :param class_type: one of [single, multi] to support multi class training;
+        default single
+    :param dropout: the amount of dropout to use while training
+    :return: The created EfficientNet_V2-S Module
+    """
+
+    sec_settings = [
+        EfficientNetSectionSettings(
+            num_blocks=2,
+            in_channels=24,
+            out_channels=24,
+            kernel_size=3,
+            expansion_ratio=1,
+            stride=1,
+            is_fused=True,
+        ),
+        EfficientNetSectionSettings(
+            num_blocks=4,
+            in_channels=24,
+            out_channels=48,
+            kernel_size=3,
+            expansion_ratio=4,
+            stride=2,
+            is_fused=True,
+        ),
+        EfficientNetSectionSettings(
+            num_blocks=4,
+            in_channels=48,
+            out_channels=64,
+            kernel_size=3,
+            expansion_ratio=4,
+            stride=2,
+            is_fused=True,
+        ),
+        EfficientNetSectionSettings(
+            num_blocks=6,
+            in_channels=64,
+            out_channels=128,
+            kernel_size=3,
+            expansion_ratio=4,
+            stride=2,
+            se_ratio=0.25,
+            se_mod=se_mod,
+        ),
+        EfficientNetSectionSettings(
+            num_blocks=9,
+            in_channels=128,
+            out_channels=160,
+            kernel_size=3,
+            expansion_ratio=6,
+            stride=1,
+            se_ratio=0.25,
+            se_mod=se_mod,
+        ),
+        EfficientNetSectionSettings(
+            num_blocks=15,
+            in_channels=160,
+            out_channels=256,
+            kernel_size=3,
+            expansion_ratio=6,
+            stride=2,
+            se_ratio=0.25,
+            se_mod=se_mod,
+        ),
+    ]
+
+    return EfficientNet(
+        sec_settings=sec_settings,
+        out_channels=1280,
+        num_classes=num_classes,
+        class_type=class_type,
+        dropout=dropout,
+        bn_kwargs={"eps": 1.0e-03},
+    )
+
+
+@ModelRegistry.register(
+    key=[
+        "efficientnet_v2_m",
+        "efficientnet_v2-m",
+        "efficientnetv2-m",
+        "efficientnetv2_m",
+    ],
+    input_shape=(3, 480, 480),
+    domain="cv",
+    sub_domain="classification",
+    architecture="efficientnet_v2",
+    sub_architecture="m",
+    default_dataset="imagenet",
+    default_desc="base",
+    def_ignore_error_tensors=["classifier.fc.weight", "classifier.fc.bias"],
+    desc_args={"optim-perf": ("se_mod", True)},
```

Comment:
maybe not for this PR, but we should consider adding some default dataloader kwargs here, so the default loaded weights are compatible with the loader

Reply:
That's a great call @bfineran